### PR TITLE
[fix][doc] Update the registry related configuration in values.yaml

### LINKED ISSUE #11434: [Doc][k8s] externalZookeeper.zookeeperRoot cannot specify the root directory of zookeeper
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

When deploying DS in a k8s cluster using Helm, the root directory of zookeeper (built-in or external) cannot be specified through `zookeeper.zookeeperRoot` or `externalZookeeper.zookeeperRoot` described in `kubernetes.md`.

截屏2022-08-11 20 10 24

In fact, `zookeeperRoot` is not referenced by any yaml file, and `_helpers.tpl` only builds `REGISTRY_TYPE` and `REGISTRY_ZOOKEEPER_CONNECT_STRING` as below:

```
{{/*
Create a registry environment variables.
*/}}
{{- define "dolphinscheduler.registry.env_vars" -}}
- name: REGISTRY_TYPE
  {{- if .Values.zookeeper.enabled }}
  value: "zookeeper"
  {{- else }}
  value: {{ .Values.externalRegistry.registryPluginName }}
  {{- end }}
- name: REGISTRY_ZOOKEEPER_CONNECT_STRING
  {{- if .Values.zookeeper.enabled }}
  value: {{ template "dolphinscheduler.zookeeper.quorum" . }}
  {{- else }}
  value: {{ .Values.externalRegistry.registryServers }}
  {{- end }}
{{- end -}}
```

So, how can user specify the root directory of zookeeper while deploying DS in k8s?

Any comments or suggestions are welcome.

### Documentation Links

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by rickchengx**
## Purpose of the pull request
close: #11434 

Update the registry related configuration in `values.yaml`.

## Brief change log

| Parameter                                                            | Description                                                                                                                   | Default                               |
|----------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|
| `zookeeper.service.port`                                             | The port of zookeeper                                                                                                         | `2181`                                |
| `externalRegistry.registryPluginDir`                                 | If exists external registry and set `zookeeper.enable` to `false`, specify the external registry plugin directory             | `lib/plugin/registry`                 |
| `externalRegistry.registryPluginName`                                | If exists external registry and set `zookeeper.enable` to `false`, specify the external registry plugin name                  | `zookeeper`                           |
| `externalRegistry.registryServers`                                   | If exists external registry and set `zookeeper.enable` to `false`, specify the external registry servers                      | `127.0.0.1:2181`                      |

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: docs/docs/en/guide/installation/kubernetes.md [hunk 1]
```diff
@@ -521,226 +521,227 @@ common:
 
 ## Appendix-Configuration
```
Comment (EricGao888) [inline docs/docs/en/guide/installation/kubernetes.md:522]: That's wired. Our Chinese docs for K8S seemed written in English?

File: docs/docs/zh/guide/installation/kubernetes.md [hunk 2]
```diff
@@ -522,226 +522,227 @@ common:
 
 ## 附录-配置
 
-| Parameter                                                                         | Description                                                                                                                    | Default                                               |
```
Comment (EricGao888) [inline docs/docs/zh/guide/installation/kubernetes.md:525]: That's wired. Our Chinese docs for K8S seemed written in English?

Comment (rickchengx) [reply docs/docs/zh/guide/installation/kubernetes.md:525]: > That's wired. Our Chinese docs for K8S seemed written in English?

Yes, but only for the `Appendix-Configuration`.

File: docs/docs/en/guide/installation/kubernetes.md [hunk 3]
```diff
@@ -521,226 +521,227 @@ common:
...
+| Parameter                                                            | Description                                                                                                                   | Default                               |
+|----------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|
+| `timezone`                                                           | World time and date for cities in all time zones                                                                              | `Asia/Shanghai`                       |
+|                                                                      |                                                                                                                               |                                       |
```
Comment (ruanwenjun) [inline docs/docs/en/guide/installation/kubernetes.md]: Is this format by spotless? If not, please remove the format change

Comment (EricGao888) [reply docs/docs/en/guide/installation/kubernetes.md]: > Is this format by spotless? If not, please remove the format change

@ruanwenjun No, we haven't enabled spotless to fix md files.

Comment (EricGao888) [reply docs/docs/en/guide/installation/kubernetes.md]: @ruanwenjun FYI https://github.com/apache/dolphinscheduler/pull/11384

Comment (rickchengx) [reply docs/docs/en/guide/installation/kubernetes.md]: > Is this format by spotless? If not, please remove the format change

This is automatically modified by `IntelliJ Idea` to a uniform table width for better document readability.

Comment (EricGao888) [reply docs/docs/en/guide/installation/kubernetes.md]: @rickchengx I've configured `Spotless` to format `markdown` automatically, see: https://github.com/apache/dolphinscheduler/pull/11384
For your convenience, you may rebase your code, make some changes in `kubernetes.md` and run `mvn spotless:apply`. In this way, `Spotless` will fix the format for you. I strongly suggest you configure the pre-commit hook following the instructions here: https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/contribute/development-environment-setup.html#code-style

Comment (rickchengx) [reply docs/docs/en/guide/installation/kubernetes.md]: > @rickchengx I've configured `Spotless` to format `markdown` automatically, see: #11384 For your convenience, you may rebase your code, make some changes in `kubernetes.md` and run `mvn spotless:apply`. In this way, `Spotless` will fix the format for you. I strongly suggest you configure the pre-commit hook following the instructions here: https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/contribute/development-environment-setup.html#code-style

@EricGao888 , Thanks for the suggestion, I've rebased and used the `spotless` to apply the code style.

**Comment by caishunfeng** [comment]
Hi @rickchengx please check the fail CI result.

**Comment by EricGao888** [comment]
> Hi @rickchengx please check the fail CI result.

@rickchengx You may rebase your code on the latest dev branch and see whether these three unused images still there. If so, just remove them and CI will pass.
image

**Comment by rickchengx** [comment]
> Hi @rickchengx please check the fail CI result.



> > Hi @rickchengx please check the fail CI result.
> 
> @rickchengx You may rebase your code on the latest dev branch and see whether these three unused images still there. If so, just remove them and CI will pass. image

Hi, @caishunfeng @EricGao888 , thanks for your suggestions! I've rebased my PR and CI is passed.

**Review by caishunfeng — APPROVED** [review]
LGTM, thanks.

**Comment by zhongjiajie** [comment]
Thanks

**Comment by zhongjiajie** [comment]
In this patch, we only want to remove two dangling configurations, but have to change other content to pass the spotless test. it will increase contributor and review time to handle the nosy, I think we should better bump all docs according to the spotless' rule, WDTY @caishunfeng @EricGao888

**Comment by caishunfeng** [comment]
> In this patch, we only want to remove two dangling configurations, but have to change other content to pass the spotless test. it will increase contributor and review time to handle the nosy, I think we should better bump all docs according to the spotless' rule, WDTY @caishunfeng @EricGao888

+1

**Comment by EricGao888** [comment]
> In this patch, we only want to remove two dangling configurations, but have to change other content to pass the spotless test. it will increase contributor and review time to handle the nosy, I think we should better bump all docs according to the spotless' rule, WDTY @caishunfeng @EricGao888 

+1, I will handle it and format all the docs once for all. Thanks for the suggestions.

**Pull request merged**